utils/imds.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html
+	IMDSv1Server      = "http://169.254.169.254"
+	DataConfigMapName = "aws-data"
```

Comment:
As this is a public const that is going to be imported in other packages please name it with a name that identifies the content
```suggestion
	IMDSDataConfigMapName = "aws-data"
```